week1/wordcount_done.py

```diff
@@ -0,0 +1,48 @@
+import sys
+def get_dictionary(filename):
+    file = open(filename, "r")
+    content = file.read()
+    words = (list(content.split()))
```

Comment:
`split()` already returns a value of type `list`. There is no need to create a new list with it.